DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: drat
 Type: Package
 Title: 'Drat' R Archive Template
-Version: 0.1.8.1
-Date: 2021-03-13
+Version: 0.1.8.2
+Date: 2021-04-08
```

Comment:
(I generally prefer to do this myself to mark releases, but I appreciate that you are also trying to set the release apart.)

Reply:
(It also prevents me from merging now as I have about the same at home :)   Quick stash will help)

Reply:
Sorry for that. 😭 

Some people like the bump, some people don't. Next time I ask.

Reply:
I am ambivalent. I like the attention to detail, and you made enough changes to mark a delta -- just how I think.  The 'cannot merge' is just a stash away so _no beans_.  The one thing I really dislike is using .9000 :)  as there is zero sense in more than one new digit, incremented starting from zero.  And while I said so for years it won't change you-know-where...

It is a bit more important in larger project / possibly use of more PRs like, say, Rcpp.  This was fine.  Thanks again.

Reply:
And the one missing piece (a `ChangeLog` entry) I now added.

Reply:
Yeah, I know. Justification is also a part of it, e.g. Why start at `.9000` and not `.0000`? And why the long number anyway? I feel it some weird calcification from a long lost regex pattern lost in time.